test/util/uk/gov/hmrc/apinotificationqueue/TestData.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apinotificationqueue
+
+import java.util.UUID
+
+import org.joda.time.DateTime
+import uk.gov.hmrc.apinotificationqueue.model.{Email, Notification, SendEmailRequest}
+import uk.gov.hmrc.apinotificationqueue.repository.{ClientNotification, ClientOverThreshold}
+
+object TestData {
+
```

Comment:
std scala constant [naming conventions](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods) pls

Reply:
Done